test/integration/battery_plugin.cc

```diff
@@ -33,7 +33,7 @@
 #include "ignition/gazebo/Server.hh"
 #include "ignition/gazebo/ServerConfig.hh"
 #include "ignition/gazebo/SystemLoader.hh"
```

Comment:
I noticed some includes in the test files aren't being changed, but they were all changed in `src`.

Reply:
Fixed, and I noticed a path that needed migration

https://github.com/gazebosim/gz-sim/compare/f6166764064941f7f713488ca9bf6041ab95d7f0..3b88c726d958c038e657b42f1c84b892b130d3a7

Path example:
https://github.com/gazebosim/gz-sim/compare/f6166764064941f7f713488ca9bf6041ab95d7f0..3b88c726d958c038e657b42f1c84b892b130d3a7#diff-69dd7ebab9966831e20afe942ffd41b3a393433781818eaf4c2319ec47d54277

Reply:
I'm still seeing a lot of `ignition/gazebo` under `test`. I think it's fine to update them in a follow up PR. I was just pointing out here that it seemed odd that some includes had been changed but not others. No action needed.

Reply:
Ah, when I was writing the script I think Steve said I didn't need to change the stuff in `src` and `test` since redirection headers were in play, so I purposely omitted anything that wasn't in a folder with `gz` anywhere in the path.

I suppose it'd get picked up in the source migration when we do a more liberal application of find and replace, I think?